shell-pipelines/ls-grep/script-03.sh

```diff
@@ -4,3 +4,11 @@ set -euo pipefail
 
 # TODO: Write a command to output the names of the files in the sample-files directory whose name starts with an upper case letter and doesn't contain any other upper case letters.
 # Your output should contain 7 files.
+ls sample-files | grep -E '^[A-Z][^A-Z]*$'
```

Comment:
What does the `-E` do here? Why do you need it here when you didn't need it for script-02 which otherwise looks quite similar?